### PR TITLE
Personal keymap for planck

### PR DESCRIPTION
**Pull request by pevecyan**

### CODE REVIEW

File: keyboards/planck/keymaps/pevecyan/config.h [hunk 1]
```diff
@@ -0,0 +1,43 @@
+#ifndef CONFIG_USER_H
```
Comment (drashna) [inline keyboards/planck/keymaps/pevecyan/config.h]: If you could replace the include guard (the ifndef, define, and endif at the end) with just `#pragma once`? 

It's simpler, and less prone to user error, so if you could make this switch?

File: keyboards/planck/keymaps/pevecyan/config.h [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+#define CONFIG_USER_H
+
+#include "config_common.h"
```
Comment (drashna) [inline keyboards/planck/keymaps/pevecyan/config.h]: This line is not actually needed, and should be removed. 

Specifically, this should be included in the keyboard's `config.h` file, and is redundant.

File: keyboards/planck/keymaps/pevecyan/keymap.c [hunk 3]
```diff
@@ -0,0 +1,256 @@
...
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "planck.h"
```
Comment (drashna) [inline keyboards/planck/keymaps/pevecyan/keymap.c]: Could you replace this include with `#include QMK_KEYBOARD_H` instead?

File: keyboards/planck/keymaps/pevecyan/keymap.c [hunk 4]
```diff
@@ -0,0 +1,256 @@
...
+  float plover_gb_song[][2]  = SONG(PLOVER_GOODBYE_SOUND);
+#endif
+
+uint32_t layer_state_set_user(uint32_t state) {
+  return update_tri_layer_state(state, _LOWER, _RAISE, _ADJUST);
+}
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+  switch (keycode) {
+    case QWERTY:
```
Comment (drashna) [inline keyboards/planck/keymaps/pevecyan/keymap.c]: If you only have this layout, then you don't really need this, and this could probably be removed.

**Comment by drashna** [comment]
Could you pull the slovenian keymap out of this PR and into a separate PR?

**Comment by pevecyan** [comment]
> Could you pull the slovenian keymap out of this PR and into a separate PR?

Sure, thanks for all suggestions. Will fix as soon as possible.

**Comment by drashna** [comment]
Thanks!

**Pull request merged**